### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by nathancooperjones**
Updating issue templates as templates GitHub will recognize. 

Since there are no code changes here, I won't be updating the version number or CHANGELOG.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+**To Reproduce**
+
+> Add a code snippet to reproduce the issue
+
+**Environment**
+
+> Add PyTorch version, OS, Python version, CUDA/cuDNN version, GPU models, etc.
+
+**Screenshots or GIFs**
```
Comment (mcundiff3) [inline .github/ISSUE_TEMPLATE/bug-report.md:26]: GIF inclusion. Amazing

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: Feature Request
+about: Suggest an idea for Collie
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (mcundiff3) [inline .github/ISSUE_TEMPLATE/feature-request.md:10]: What if it's not should this section still get filled out?

**Review by mcundiff3 — COMMENTED** [review]
Minor comment on the feature addition. I'm so excited for Collie!!!!

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (nathancooperjones) [reply .github/ISSUE_TEMPLATE/feature-request.md:10]: I think then it's okay to just leave this section blank or write `N/A`.

**Review by mcundiff3 — APPROVED** [review]
LGTM!

**Pull request merged**